airbyte-webapp/public/index.html

```diff
@@ -13,6 +13,12 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="stylesheet" href="%PUBLIC_URL%/index.css">
     <title>Airbyte</title>
+    <% if (process.env.REACT_APP_OSANO) { %>
+      <style>
+        .osano-cm-widget { display: none; }
```

Comment:
:information_source: This is the official way to hide their widget (cookie button on the bottom of the page): https://docs.osano.com/hiding-the-cookie-widget